src/elements/02-elements/shortcut/shortcut.hbs

```diff
@@ -0,0 +1,8 @@
+<a href="#" class="ys-shortcut" title="Music">
```

Comment:
What's the approach when it comes to `title` attributes? I can't see them used elsewhere in DNA.

Reply:
It is because of truncation. It we need to truncate a long text, it is a nice feature to make the text hover able and show the entire text. But this shortcut does not have a limit when it comes to text. So it's actually not used for anything. I will remove the title attribute again.